gromacs/__init__.py

```diff
@@ -242,6 +242,17 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+# Try to load environment variables setted by GMXRC
+gmxrc = config.cfg.get("Gromacs", "GMXRC")
+if not gmxrc:
```

Comment:
... Ignore my last comment regarding the `cfg.get()`, I just realized you had already added a default of "".